libs/application/templates/party-letter/README.md

```diff
@@ -1,10 +1,10 @@
 # Party Letter
 
-This application allows governmental parties to get assigned a party letter for upcoming elections
+This application allows governmental parties to get assigned a party letter for upcoming elections.
 
 ## Setup
 
-Application System setup: [Setup](https://github.com/island-is/island.is/tree/main/apps/application-system)
+Application System setup: [Setup](https://docs.devland.is/handbook/apps/application-system)
```

Comment:
Noticed that the URL is now https://docs.devland.is/apps/application-system (without "handbook" in the path)